test/visualization/plotting.jl

```diff
@@ -1,35 +0,0 @@
-### Not currently run by runtests
-
```

Comment:
These tests where not run, and are superseded by new indexing tests, which does this but are also more extensive